src/app/saga.ts

```diff
@@ -291,6 +292,10 @@ export function* handleDeepLink(action: OpenDeepLink) {
       // of our own notifications for security reasons.
       const params = convertQueryToScreenParams(rawParams.query)
       navigate(params.screen as keyof StackParamList, params)
+    } else if (pathParts[1] === 'share' && pathParts.length === 3) {
```

Comment:
should we swap the order of this condition, so that we ensure `pathParts[1]` is not going to cause issues in case the length is 0?

Reply:
Hmmm - `undefined === 'share'` shouldn't throw, if that's what you're thinking. I'm happy to swap it though.